### DISCUSSION TITLE
Idea for Small Modification in B2B apis payload for NP's.

### DISCUSSION THREAD

**Discussion by SignCatch-NP** [ONDC-Official/ONDC-RET-Specifications#50] [General]
We at SignCatch request a small payload change from the buyersNP that each endpoint /select, /init, /confirm. should contain the information which is the descriptor of the buyer which includes the name(mandatory), logo(optional), and address(mandatory) of his business by the buyerNP.

Why is this change important?
In the current scenario, the buyer requests for the quotation, and through NP /select, a request is made to sellerNP and the request doesn't have such information(name, logo) to share with the seller which keeps them blind like who is the front party while the seller is sharing the quotation. 

In B2B the Business info for the parties is necessary like who is the front party who is requesting the quote from the seller this info is necessary to build and easy to maintain the relationship with knowing the customer, this small change in interfaces brings trust, and comfort to the seller enhancing the overall UX for the sellers.

**Comment by tanyamadaan** [comment]
As discussed with everyone, in order to provide better user experience and to be able to validate the buyer tax id with the company trade name, we are proposing to receive the company trade name as well as part of the RFQ request.

```
"message": {
    "order": {
      "fulfillments": [
        {
          "id": "F1",
          "type": "Delivery",
          "stops": [
            {
              "type": "end",
              "location": {...}
            }
          ],
          "customer": {
            "organization": {
              "descriptor": {
                "name": "abc pvt limited"
              },
              "creds": [
                {
                  "id": "ESG-12345678",
                  "type": "License",
                  "desc": "License No. ESG-12345678",
                  "icon": "https://abcd.cdn.com/images/license-img",
                  "url": "https://abcd.dnb.com/verify?id=’ESG-12345678'"
                }
              ]
            }
          }
        }
      ]
    }
  }
```

**Reply by ashok-ai** [reply]
@tanyamadaan Are we implementing these changes? We are also facing the same issue and we needed this customer name to be displayed in the app.
